src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -229,6 +230,21 @@ public final HttpServer host(String host) {
 		return super.host(host);
 	}
 
+	/**
+	 * Apply HTTP/2 configuration
+	 *
+	 * @param http2Settings configures {@link Http2Settings} before requesting
+	 * @return a new {@link HttpServer}
+	 */
+	public final HttpServer http2Setting(Consumer<Http2Settings> http2Settings) {
```

Comment:
We should choose a constitent way to expose builders, most of the code use spec, I think this will be a unique one. WDYT

Reply:
That one in particular I want to discuss because at the moment `Http2Settings` is an API provided by Netty. Do we want to use this directly from Netty or it is better to wrap it with our API.

Reply:
I added `Http2SettingsSpec`